mm2src/mm2_main/src/lp_swap/taker_swap_v2.rs

```diff
@@ -441,6 +441,17 @@ impl<MakerCoin: MmCoin + MakerCoinSwapOpsV2, TakerCoin: MmCoin + TakerCoinSwapOp
             SecretHashAlgo::SHA256 => sha256(self.taker_secret.as_slice()).take().into(),
         }
     }
+
+    fn dex_fee(&self) -> DexFee {
+        let dummy_unique_data = vec![];
+        let taker_pub = self.taker_coin.derive_htlc_pubkey_v2_bytes(&dummy_unique_data); // Using dummy swap data because we need only permanent taker pubkey for dex fee
```

Comment:
nit: i see that this eventually gets our public key of the keypair and the dummy/swap uniqe data is never used.
i feel we should have a method to get the pubkey of taker coin cuz that look confusing.

Reply:
unique_data used for zcoin only.
Here for dexfee calc we dont use zcoin temp pubkey, only permanent pubkeys
added taker_pubkey_bytes fn in 56f13b2077664cec8f3d0ecbfdfc286e002d5b38